FV3/makefile

```diff
@@ -24,8 +24,45 @@ PPSER_FLAGS = --verbose --ignore-identical -m utils_ppser_kbuff
 FV3_EXE  = fv3.exe
 FV3CAP_LIB  = libfv3cap.a
 
+PWD :=$(shell pwd)
+TOP_LEVEL_OBJS_RELATIVE = atmos_model.o module_fv3_config.o
+TOP_LEVEL_OBJS_ABSOLUTE = $(addprefix $(PWD)/, $(TOP_LEVEL_OBJS_RELATIVE))
+
+# Linking flags
+FV3_LIB_DIRS = atmos_cubed_sphere io ipd gfsphysics  ../stochastic_physics cpl coarse_graining
+FV3_LIBS_ABS = $(addprefix -L$(PWD)/, $(FV3_LIB_DIRS))
+FV3_LIB_FLAGS = -lfv3core -lfv3io -lipd -lgfsphys -lstochastic_physics -lfv3cpl -lfv3coarse_graining $(FV3_LIBS_ABS)
+
+ifeq ($(AI2_WRAPPER),Y)
+	MAYBE_WRAPPER_TARGET = wrapper_build
+endif
```

Comment:
I don't understand what the AI2_WRAPPER logic is doing in practice. From my reading:
- it is set in configure.fv3, and in that context it's unclear what it does - kind of seems like an unused variable
- it's set in the nix shell, unclear what the purpose of this is since it's already set in configure.fv3 used by nix
- It's read only in this file, and in this file it only affects what `make all` does, but we never call `make all` (or bare `make` which triggers all) as far as I can tell

It looks like this flag makes nix build the wrapper (before libs - doesn't the wrapper depend on libs? That's also confusing to me)? Though I don't understand how it accomplishes this.

If I understand the purpose of this logic, can you replace it with an explicit `make wrapper_build` somewhere in the nix build instructions?